toc.yml

```diff
@@ -444,8 +444,8 @@
           topicHref: /dotnet/csharp/programming-guide/concepts/index
           items:
           - name: Assemblies
-            tocHref: /dotnet/csharp/programming-guide/concepts/assemblies-gac/
-            topicHref: /dotnet/csharp/programming-guide/concepts/assemblies-gac/index
+            tocHref: /dotnet/standard/assembly/
```

Comment:
This should be moved completely out of the C# section of the TOC to the .NET Standard section of the TOC. It's causing all of the .NET Standard assembly documentation to appear with C# breadcrumbs.

Reply:
THANK YOU! I couldn't find it...